daemon/algod/api/algod.oas2.json

```diff
@@ -2303,9 +2303,27 @@
           },
           {
             "type": "integer",
-            "description": "Max number of box names to return. If max is not set, or max == 0, returns all box-names.",
+            "description": "Maximum number of boxes to return. Server may impose a lower limit.",
             "name": "max",
             "in": "query"
+          },
+          {
+            "type": "string",
+            "description": "A box name prefix, in the goal app call arg form 'encoding:value'. For ints, use the form 'int:1234'. For raw bytes, use the form 'b64:A=='. For printable strings, use the form 'str:hello'. For addresses, use the form 'addr:XYZ...'.",
+            "name": "prefix",
+            "in": "query"
+          },
+          {
+            "type": "string",
```

Comment:
Slightly confused by the description, maybe will be more clear when reading the supplying code. From reading the code, it looks like the next-token is in fact the name of the next box.

Reply:
I tried to describe what it does, though perhaps I shouldn't.  If you use the next-token supplied, without understanding what it "means", you'll be fine.  I documented it so that someone could, in principle, use it even if they weren't doing pagination.  If you want boxes that start with "xx", but you want to start at "xxJ", you can.

There's another school of thought that says the next-token is a black box, and the `next` parameter can take what's given and nobody should try to understand it.